.circleci/config.yml

```
@@ -155,13 +183,21 @@ jobs:
             adb shell settings put global transition_animation_scale 0.0
             adb shell settings put global animator_duration_scale 0.0
       - *adb_logging
-      - *build_test_android
+      - run:
+          name: Run android unit test
```

```suggestion
          name: Run instrumented test on android emulator
```

Actually we run instrumented test on Android emulator not unit tests.

---

@aeharvlee applied